### PR TITLE
docs(README): Fix up package managers example

### PR DESCRIPTION
**Pull request by tsteenbe**
Fix up `ort-scan` variable used to set which package managers to enable in a ORT run.
This used to be ORT Analyzer specific command but has since been changed to ORT CLI parameter.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -89,7 +89,7 @@ ort-scan:
     VCS_URL: 'ssh://git@github.com:jshttp/mime-types.git'
     VCS_REVISION: 'ef932231c20e716ec27ea159c082322c3c485b66'
     ALLOW_DYNAMIC_VERSIONS: 'true'
-    ORT_CLI_ANALYZE_ARGS: '-P ort.analyzer.enabledPackageManagers=NPM,Yarn,Yarn2'
+    ORT_CLI_ARGS: '-P ort.analyzer.enabledPackageManagers=NPM,Yarn,Yarn2 -P ort.forceOverwrite=true --stacktrace'
```
Comment (fviernau) [inline README.md:92]: Why did you add `-P ort.forceOverwrite=true --stacktrace` ?

Comment (tsteenbe) [reply README.md:92]: I do this to retain the default value for ORT_CLI_ARGS see https://github.com/oss-review-toolkit/ort-ci-gitlab/blob/main/templates/ort-scan.yml#L82

Comment (fviernau) [reply README.md:92]: It'd be nice to either mention in the commit message or to make it a separate commit IMHO.

Comment (tsteenbe) [reply README.md:92]: Opted for updating the commit message

**Review by fviernau — APPROVED** [review]
